src/errors/version-hint-message.ts

```diff
@@ -0,0 +1,3 @@
+export function versionErrorHintMessage(message: string, method: string) {
+  return `${message}\nHint: It might not be working because you're not up to date with the Meilisearch version that ${method} call requires.`
```

Comment:
I've added to the ruby method the word "maybe".

```suggestion
  return `${message}\nHint: It might not be working because maybe you're not up to date with the Meilisearch version that ${method} call requires.`
```
But I'm still don't know if this message is clear enough 🤔